src/app/portlets/dot-edit-page/layout/components/dot-edit-layout-grid/dot-edit-layout-grid.component.ts

```diff
@@ -1,14 +1,25 @@
-import { Component, OnInit, forwardRef, ViewChild } from '@angular/core';
+import { Component, OnInit, forwardRef, ViewChild, OnDestroy } from '@angular/core';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR, FormGroup, FormBuilder } from '@angular/forms';
+
 import * as _ from 'lodash';
+import { NgGrid, NgGridConfig } from 'dot-layout-grid';
+
+import { DOT_LAYOUT_GRID_MAX_COLUMNS } from './../../../shared/models/dot-layout-grid.model';
```

Comment:
use relative paths ?

Reply:
This shared is inside the layout folder is not the same as `@models`